Userland/Libraries/LibGfx/EdgeFlagPathRasterizer.cpp

```diff
@@ -51,12 +52,17 @@ static Vector<Detail::Edge> prepare_edges(ReadonlySpan<FloatLine> lines, unsigne
         if (p0.y() == p1.y())
             continue;
 
-        auto dx = p1.x() - p0.x();
-        auto dy = p1.y() - p0.y();
-        float dxdy = float(dx) / dy;
-        float x = p0.x();
+        auto min_y = static_cast<int>(p0.y());
+        auto max_y = static_cast<int>(p1.y());
```

Comment:
maybe use `truncf` instead of going from float to int to float again

Reply:
`min_y` and `max_y` and integers when stored in the `Details::Edge`, so I think it makes sense to keep them integers here (this part of the code not that hot anyway).